### PR TITLE
Added some details of content types and taxonomy

### PR DESCRIPTION
**Pull request by Yemaneberhan-Lemma**

### CODE REVIEW

File: docs/content_structure.md [hunk 1]
```diff
@@ -4,20 +4,73 @@
...
+- **Description**: Use the Event contributions content type to document contributions made during events, including presentations, workshops, or collaborative efforts.
+- **Fields used**:
+  - Active Event (field_event_active): Indicates whether the event is currently active or inactive (Boolean).
+  - Description (body): A detailed, formatted text field with a summary to describe the event.
+  - Event Additional Links (field_event_additional_links): URLs for additional information or resources related to the event.
+  - Event Address (field_event_address): The physical address where the event will take place.
+  - Event Dates (field_event_dates): The start and end dates (and times) for the event.
+  - Event Link (field_event_link): A primary URL link associated with the event.
+  - Event Location (field_event_location): Geographical coordinates (latitude and longitude) of the event’s location.
+  - Event Type (field_event_contrib_event_type): References the type of event, categorized by terms from the Event Type taxonomy.
```
Comment (hussainweb) [inline docs/content_structure.md:45]: All these are fields on the Event content type, not Event contribution.

**Review by hussainweb — CHANGES_REQUESTED** [review]

File: docs/content_structure.md (hunk 1 above)
Comment (Yemaneberhan-Lemma) [reply docs/content_structure.md:45]: @hussainweb: It appears like that, but the reality is - those fields are practically under the Event Contribution content type.

Comment (Yemaneberhan-Lemma) [reply docs/content_structure.md:40]: > How do you mean? When creating an event contribution, all you can do is select an existing event. You cannot touch any of the fields mentioned here when creating an event contribution.

@hussainweb: Thank you for the clarification and I made mistake by switching fields of Events with Event Contributions. Now, I fixed it. Please review.

**Review by hussainweb — APPROVED** [review]
Thanks for your work here, @Yemaneberhan-Lemma. Further, we should add why these content types exist. How is someone meant to use this website through these content types? That should be in another PR.

**Pull request merged**